programs/zstdcli.c

```diff
@@ -77,6 +77,7 @@ static const unsigned g_defaultMaxDictSize = 110 KB;
 static const int      g_defaultDictCLevel = 3;
 static const unsigned g_defaultSelectivityLevel = 9;
 static const unsigned g_defaultMaxWindowLog = 27;
+static const double   g_defaultDictRegression = 1.01;
```

Comment:
Add support for this.

We can't accept doubles on the command line. Instead, can we take a percent 0, 1, etc?